Spec/Auth.swift

```diff
@@ -1692,7 +1689,7 @@ class Auth : QuickSpec {
                             guard let tokenDetails = tokenDetails else {
                                 XCTFail("TokenDetails is nil"); done(); return
                             }
-                            expect(tokenDetails.token).to(equal(options.token))
+                            expect(tokenDetails.token).toNot(equal(options.token))
```

Comment:
The comment above says `Reuse the valid token` yet that's not correct.  Also, I would like to see that implicit publish operations don't now create a new token every time.  Could we therefore change the logic for this test to:

* Publish a message
* Check that token exists
* Publish another message
* Check that token has not changed
* Call authorize
* Check that token has changed